treeherder/perf/auto_perf_sheriffing/backfill_reports.py

```diff
@@ -53,7 +53,8 @@ def extract_important_alerts(self, alerts: Tuple[PerformanceAlert, ...]):
         if any(not isinstance(alert, PerformanceAlert) for alert in alerts):
             raise ValueError('Provided parameter does not contain only PerformanceAlert objects.')
         relevant_alerts = self._extract_by_relevant_platforms(alerts)
-        sorted_alerts = self._multi_criterion_sort(relevant_alerts)
+        alerts_with_dist_jobs = self._ensure_distinct_jobs(relevant_alerts)
```

Comment:
Nit: I think it'd be helpful to rename this `alerts_with_distinct_jobs` since the meaning of 'dist' is a bit ambiguous.